app/views/NirmsQuestionView.scala.html

```diff
@@ -0,0 +1,52 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    heading1: components.Heading1,
+    heading2: components.Heading2,
+    paragraph: components.Paragraph,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton,
+    link: components.Link
+)
+
+@(form: Form[Boolean])(implicit request: Request[_], messages: Messages)
+
+@layout(
+    pageTitle    = titleNoForm(messages("nirmsQuestion.title")),
+    showBackLink = true
+) {
+    @heading1(messages("nirmsQuestion.h1"))
+    @paragraph(Html(messages("nirmsQuestion.p1")))
+    @paragraph(Html(messages("nirmsQuestion.p2", link(messages("nirmsQuestion.p2.linkText"),"https://www.gov.uk/government/publications/retail-movement-scheme-how-the-scheme-will-work/retail-movement-scheme-how-the-scheme-will-work#product-eligibility"))))
```

Comment:
could we split this over 2 lines for readability

Reply:
Done